Bio/FSSP/fssp_rec.py

```diff
@@ -1,4 +1,7 @@
-# A superclass for reading [f]ixed-column type [f]lat-[f]ile records. (e.g.
+'''A superclass for reading [f]ixed-column type [f]lat-[f]ile records.'''
```

Comment:
We tend to use triple double quotes in Biopython, rather than triple single quotes.